src/screens/SearchScreen/components/DictionaryEntriesContainer.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {Text, ScrollView} from 'react-native';
+import PropTypes from 'prop-types';
+import DictionaryEntry from './DictionaryEntry.js';
+
+const getEntries = (data, emptyMessage) => {
+  if (emptyMessage.length != 0) return <Text>{emptyMessage}</Text>;
```

Comment:
We also talked about this - I think in this case the code here is simple enough to just go into the component below instead.